lib/pycoproc/pycoproc.py

```diff
@@ -245,7 +255,7 @@ def calibrate_rtc(self):
         self.i2c.deinit()
         Pin('P21', mode=Pin.IN)
         pulses = pycom.pulses_get('P21', 100)
-        self.i2c.init(mode=I2C.MASTER, pins=(self.sda, self.scl), baudrate=100000)
+        self.i2c.init(mode=I2C.MASTER, pins=(self.sda, self.scl))
```

Comment:
baudrate

Reply:
as Christian mentioned, it's no difference